scripts/update-api-tokens.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# If running locally you will need to provide AWS_PROFILE=sops or equivalent.
+# If running locally through Docker, you will probably also need to mount your ~/.aws folder at `/root/.aws` for SOPS to be able to decrypt credentials. But be careful with this due to file permissions.
+# If running on Jenkins, AWS allows SOPS to decrypt credentials using its EC2 instance profile - even within Docker.
+
+# Syntax: ./scripts/update-api-tokens.sh [add-new|remove-old] [preview|staging|production]
+
+set -e
+
+COMMAND=$(echo "${1}" | tr "[:upper:]" "[:lower:]")
+STAGE=$(echo "${2}" | tr "[:upper:]" "[:lower:]")
+GIT_CREDS_UPDATE_BRANCH_NAME=$(date +"%Y-%m-%dT%k-%M-%S-credentials-update")
+
+if [ -z ${GITHUB_ACCESS_TOKEN} ]; then
+  echo "Required environment variable GITHUB_ACCESS_TOKEN is undefined."
+  exit 1
+fi
+
+if [[ "${STAGE}" != "preview" && "${STAGE}" != "staging" && "${STAGE}" != "production" ]]; then
+  echo "Syntax: ./scripts/update-api-tokens.sh [add-new|remove-old] [preview|staging|production]"
+  exit 1
+fi
+
+function generate_api_token() {
+  echo $(python3 -c "import random, string; print(''.join(random.SystemRandom().choice(string.ascii_letters + string.digits) for _ in range(56)))")
+}
+
+function create_credentials_update_branch() {
+  rm -rf digitalmarketplace-credentials
```

Comment:
Should we do something like .gitignore `digitalmarketplace-credentials` or get this script to remove it after it's run?

Reply:
Hm interesting. We could do - although in theory this should only run inside a Docker container, so it won't be left behind anyway. If a developer runs this locally outside of Docker, i guess it could get left around, but that shouldn't be done and in that case I'm kinda happy leaving it there to spite them for doing it improperly... 🙈